### PR TITLE
move get-configvals to runtime subrpm

### PR DESCRIPTION
**Pull request by DrDaveD**
**Description of the Pull Request (PR):**

This moves get-configvals, introduced in #1082, to the runtime subrpm.  Otherwise if only singularity-runtime rpm is installed, doing something like a singularity exec causes these errors:
```
/usr/libexec/singularity/cli/action_argparser.sh: line 212: /usr/libexec/singularity/bin/get-configvals: No such file or directory
/usr/libexec/singularity/cli/action_argparser.sh: line 212: [: ==: unary operator expected
```

**This fixes or addresses the following GitHub issues:**

- Ref: None

**Checkoff for all PRs:**

- [x] I have read the [Guidelines for Contributing](https://github.com/singularityware/singularity/blob/master/CONTRIBUTING.md), and this PR conforms to the stated requirements.
- [ ] I have added changes to the [CHANGELOG](https://github.com/singularityware/singularity/blob/development/CHANGELOG.md) and and documentation updates to the [singularityware](https://www.github.com/singularityware/singularityware.github.io) documentation base.
  Not needed because this fixes an unreleased bug.
- [ ] I have tested this PR locally with a `make test` 
   Not appropriate because it only affects rpms
- [x] This PR is NOT against the project's master branch
- [x] I have added myself as a contributor to the [contributors's file](https://github.com/singularityware/singularity/blob/master/CONTRIBUTORS.md)
- [x] This PR is ready for review and/or merge

Attn: @singularityware-admin

### CODE REVIEW

**Comment by GodloveD** [comment]
Hi @DrDaveD.  Thanks very much for this PR and thanks too for your participation in our hackathon last week.  As we discussed there, we are now working on the `development-3.0` branch (though that name may change).  I don't think we know how the packaging is going to look once the code is rewritten in golang.  I'm going to leave this open so we don't forget about the issue.  But I think we should hold off on merging until we have some more development finished on the golang stuff.

**Comment by DrDaveD** [comment]
@GodloveD I thought we agreed that the main development branch should proceed with accepting PRs until the 3.0 feature branch was at a stable enough state.  Then we would freeze the development branch for a short time while the feature branch caught up with recent changes in development, then merge it in.   

So, I don't think it is the right thing to do to hold off merging all PRs into the development branch that might be done differently once the future feature branch is merged.  It could take months, and for all we know, this PR may be directly relevant.  The main development branch should continue to move forward in the meanwhile; we can't hold off all other development that might possibly have to be merged into the feature branch at some point, or much other progress will come to halt.

For this particular PR, holding off merging it is going to block addressing issue #1326.  I don't want to wait for some unspecified amount of time to address that.

**Comment by GodloveD** [comment]
Oh yes! You are totally correct. Sorry that I did not remember the final decision that we arrived at.  OK, then.  Let me have a closer look at this and your other recent PRs and I'll let you know.  I also saw you request in Slack that they be included in the 2.4.3 release candidate.  I'm happy to do that too as long as no one else objects.

**Comment by DrDaveD** [comment]
This particular PR I think only applies to the development branch because it relates to a new as of yet unreleased feature.

**Comment by GodloveD** [comment]
Indeed.  Thanks.

**Comment by DrDaveD** [comment]
I rebased against the current development branch.  Even though this is a dead-end branch, I presume that the changes that added get-configvals will one day be ported to the other branches so I would think this PR is still relevant.

**Comment by DrDaveD** [comment]
Closing and reopening to re-run travis.

**Comment by DrDaveD** [comment]
Nevermind about travis, there's something wrong with it in the development branch where it attempts to install a zypper package.

**Comment by DrDaveD** [comment]
Rebased to development-2.x because of #1681.  @GodloveD this is important, it is a very ugly error message when only singularity-runtime is installed.

**Comment by DrDaveD** [comment]
I'm going to close & reopen this to trigger a travis build.

**Review by GodloveD — APPROVED** [review]
Good catch.  Thanks.

**Review by jmstover — APPROVED** [review]

**Comment by GodloveD** [comment]
Thanks @DrDaveD and @jmstover

**Pull request merged**
